raiko-guest/src/one_shot.rs

```diff
@@ -42,14 +42,24 @@ pub async fn one_shot(global_opts: GlobalOpts, args: OneShotArgs) -> Result<()>
     .unwrap();
     println!("Reading input file {} (block no: {})", path_str, block_no);
 
-    // TODO FIXME
-    let block_header_hash_str = String::from("foobar");
-    // let block_header_hash_str = get_data_to_sign(path_str, block_no).await?;
-    println!("Data to be signed: {}", block_header_hash_str);
+    let (_next_private_key, next_public_key, secp) = generate_new_keypair(secrets_dir)?;
+    // TODO: save next private key
```

Comment:
It is saved in here:
https://github.com/johntaiko/zeth/blob/f7cde1178ee159ebe873fa8cc634e4115912942b/raiko-guest/src/one_shot.rs#L90